src/styles/index.css

```diff
@@ -0,0 +1,93 @@
+@import "./stars.css";
+
+@import "./variables.css";
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 166px;
```

Comment:
use `box-sizing: border-box;`  (in * selector) and set the final width ('200px') instead of calculating it